best_practices/controllers.rst

```diff
@@ -13,8 +13,8 @@ into a service.
 
 .. best-practice::
 
-    Make your controller extend the ``FrameworkBundle`` base Controller and
-    use annotations to configure routing, caching and security whenever possible.
+    Make your controller extend the FrameworkBundle base Controller and use
```

Comment:
You can also lowercase Controller (or put it in literals if we use it as the class name here).